### PR TITLE
Actual 1st seasonal. definitely not a lie.

### PR DESCRIPTION
**Pull request by ghost**
WIP

### CODE REVIEW

**Comment by ghost** [comment]
Despite what the Travis build says, it's still a wip

File: mods/seasonal/scripts.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+exports.BattleScripts = {
+  //Credits to spandan for the code for metaman so i can use this for reasons
+	pokemon:{
```
Comment (DeathlyPlays) [inline mods/seasonal/scripts.js]: Space after ``pokemon:`` imo

File: mods/seasonal/scripts.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+exports.BattleScripts = {
+  //Credits to spandan for the code for metaman so i can use this for reasons
```
Comment (DeathlyPlays) [inline mods/seasonal/scripts.js]: Define a "randomCountryTeam"

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: mods/seasonal/scripts.js (hunk 2 above)
Comment (ghost) [reply mods/seasonal/scripts.js]: I will

File: mods/seasonal/scripts.js (hunk 1 above)
Comment (ghost) [reply mods/seasonal/scripts.js]: K

File: mods/seasonal/scripts.js (hunk 2 above)
Comment (DeathlyPlays) [reply mods/seasonal/scripts.js]: I would add that code but not a collaborator lol

Comment (DeathlyPlays) [reply mods/seasonal/scripts.js]: `randomCountryTeam: function (side) {`

Comment (ghost) [reply mods/seasonal/scripts.js]: Ty

**Comment by ghost** [comment]
Feel free to add pokemon sets to the meta, but they have to be based off of a country!

**Comment by ghost** [comment]
@HoeenCoder Close this PR

**Comment by HoeenCoder** [comment]
Why not just start changing your name and not deleting your account?

**Comment by ghost** [comment]
Sorry

**Pull request closed without merging**